pkgs/applications/misc/remarkable/reStream/default.nix

```diff
@@ -0,0 +1,55 @@
+{ lib
+, bash
+, stdenv
+, lz4
+, ffmpeg-full
+, fetchFromGitHub
+, openssh
+, makeWrapper
+}:
+
+stdenv.mkDerivation rec {
+  pname = "reStream";
+  version = "1.1";
+
+  src = fetchFromGitHub {
+    owner = "rien";
+    repo = pname;
+    rev = version;
+    sha256 = "18z17chl7r5dg12xmr3f9gbgv97nslm8nijigd03iysaj6dhymp3";
+  };
+
+  nativeBuildInputs = [ makeWrapper ];
+  buildInputs = [ bash ];
+
+  dontConfigure = true;
+  dontBuild = true;
+
+  installPhase = ''
+    runHook preInstall
+
+    mkdir -p $out/{bin,share}
+
+    # copy the reMarkable ARM binary to the output directory
+    cp ${src}/restream.arm.static $out/share
+
+    # copy the client script to the output directory
+    cp ${src}/reStream.sh $out/bin/reStream
```

Comment:
In the AUR, the executable has a different name: https://aur.archlinux.org/cgit/aur.git/tree/PKGBUILD?h=restream-git
Also, `install` is better here.